gradle/versions.gradle

```diff
@@ -64,5 +68,9 @@ dependencyManagement {
     dependency 'org.web3j:core:4.5.14'
 
     dependency 'org.zeroturnaround:zt-exec:1.11'
+
+    dependency 'info.picocli:picocli:4.5.1'
+
+    dependency 'tech.pegasys.teku.internal:bls:0.12.6'
```

Comment:
don't think Signers can depend on Teku - this feels a bit backwards.

Reply:
it is used in tests.

Reply:
but since the data being stored doesn't need to be BLS, this can be removed.

Reply:
removed.